appcues/src/main/java/com/appcues/data/MoshiConfiguration.kt

```diff
@@ -55,7 +56,8 @@ internal object MoshiConfiguration {
         .add(SerializeNull.JSON_ADAPTER_FACTORY)
         .add(StepContainerAdapter())
         .add(ExperienceStepFormStateAdapter())
-        .add(KotlinJsonAdapterFactory())
+        .add(LossyExperienceResponseAdapter())
+        .addLast(KotlinJsonAdapterFactory())
```

Comment:
changed to `addLast` per documentation note https://github.com/square/moshi#reflection